components/TheHeader.vue

```diff
@@ -13,27 +13,38 @@ const route = useRoute()
 const isThemeDialogOpened = ref(false)
 const isLangDialogOpened = ref(false)
 const isMenuOpened = ref(false)
+const isToggleOpened = ref(false)
 
 const toggleThemeDialog = () => {
   isMenuOpened.value = false
   isLangDialogOpened.value = false
   isThemeDialogOpened.value = !isThemeDialogOpened.value
+  isToggleOpened.value = false
 }
 const toggleLangDialog = () => {
   isThemeDialogOpened.value = false
   isMenuOpened.value = false
   isLangDialogOpened.value = !isLangDialogOpened.value
+  isToggleOpened.value = false
 }
 const toggleMenu = () => {
   isThemeDialogOpened.value = false
   isLangDialogOpened.value = false
+  isToggleOpened.value = false
   isMenuOpened.value = !isMenuOpened.value
 }
+const toggleHamburgerMenu = () => {
```

Comment:
What is this? Can you explain in detail.

Reply:
To toggle mobile menu by clicking hamburger menu